.github/workflows/python-deps.yml

```diff
@@ -28,9 +28,11 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Install prerequisites
-        # brew can be finicky but it does not always means that the rest of the job will fail
+        env:
+          HOMEBREW_NO_AUTO_UPDATE: on
+        # Necessary packages for some Python deps
         run: |
-          make prerequisites
+          brew install libxmlsec1
```

Comment:
Why not run `make prerequisites`?

Reply:
`brew bundle` includes a lot of packages that could suddenly be upgraded and start failing.
Reducing it to `libxmlsec1` reduces it to only one package.

I want the `python-deps` to fail the least to avoid engineers be confused.
It is also *almost* what we used to do before `make prerequisites` https://github.com/getsentry/sentry/commit/3f71db8928bf6e20eb821026ad6713b4b043f2bf#diff-9bec90e4ad9a179ec1c8e1266c2ddc2fc0fbd73db55720b3425cfdf667b89e14